src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
It seems like you're trying to link a SCSS file directly in your HTML. Browsers don't understand SCSS code, only CSS. You need to compile your SCSS to CSS and then link the compiled CSS file here.